.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+dist: trusty
+language: cpp
+compiler:
+  - gcc
+  - clang
+env:
+  - BUILD_TYPE=Debug
+  - BUILD_TYPE=Release
```

Comment:
In Linux, it is probably more useful to setup RelWithDebInfo instead of just Release. That is the default in debian.